### DISCUSSION TITLE
Some games (Sims & Overwatch) don't go full screen on i3

### DISCUSSION THREAD

**Discussion by SpaceLarry14** [i3/i3#5877] [Q&A]
Hey Guys,

New to the i3 Window Manager and loving everything so far, been successful at installing Picom and ricing my system etc, so not afraid to screw around with conf files.

However when playing a small minority of games with the Wine compatibility layer, I cannot get the game to run in full screen, the i3 full screen command makes it "go full screen" but then there's a section of the screen cut off. If i use ALT-Enter, I get windowed full screen, but this has huge performance penalties.

Any solutions for this? Image attached
![2024-01-21-13:14:34-screenshot](https://github.com/i3/i3/assets/155795793/a98092b6-5949-4b76-9503-6c39646c119b)

**Comment by mangosherbet** [comment]
Not sure if this is the solution you're look for but in your i3 config file you can set the window class of your applications to be full screen.  Just make sure to change the .* with the window class name.

```
for_window [class=".*"] fullscreen enable
```

You can find the window classes by using the command `xprop`
